src/UI/Handlebars/Helpers/Html.js

```diff
@@ -14,15 +14,24 @@ window.NzbDrone.imageError = function(img) {
 };
 
 Handlebars.registerHelper('defaultImg', function(src, size) {
+    var endOfPath = /\.jpg($|\?)/g;
+    var errorAttr = 'onerror="window.NzbDrone.imageError(this);"';
+    var srcsetAttr = '';
+    var oneX, twoX;
+
     if (!src) {
-        return new Handlebars.SafeString('onerror="window.NzbDrone.imageError(this);"');
+        return new Handlebars.SafeString(errorAttr);
     }
 
     if (size) {
-        src = src.replace(/\.jpg($|\?)/g, '-' + size + '.jpg$1');
+        oneX = src.replace(endOfPath, '-' + size + '.jpg$1');
+        twoX = src.replace(endOfPath, '-' + size * 2 + '.jpg$1');
+        srcsetAttr = 'srcset="{0} 1x, {1} 2x"'.format(oneX, twoX);
     }
 
-    return new Handlebars.SafeString('src="{0}" onerror="window.NzbDrone.imageError(this);"'.format(src));
+    return new Handlebars.SafeString(
+        'src="{0}" {1} {2}'.format(src, srcsetAttr, errorAttr)
```

Comment:
D'oh. I knew there was a reason for `oneX = src`. Are there any tests for these helpers I should add to? Or is this maintenance-mode until the new UI comes out?